CHANGELOG.md

```diff
@@ -4,6 +4,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.3] - TBD
+
+### Added
+- Sharded Grad Scaler works with cpu offload in mixed and full precision. [#831]
+- Chore: `Pre-Commit` will be the de-facto standard to run all linter checks (black, isort, mypy, flake8 etc.)
```

Comment:
I would skip this since we already had a precommit config before this PR. The only change is that it will be the recommended way forward.